build.sbt

```diff
@@ -56,7 +55,7 @@ def baseSettings: Seq[Setting[_]] =
 //   minimalSettings ++ baseScalacOptions ++ Licensed.settings ++ Formatting.settings
 
 def addBaseSettingsAndTestDeps(p: Project): Project =
-  p.settings(baseSettings).configure(addTestDependencies)
+  p.settings(baseSettings).configure(addTestDependencies _)
```

Comment:
Thanks for the heads up Dale, I didn't know this and yeah, it's probably what happened. I've just updated the PR. It's a shame scalac behaves this way, hope this changes.